addons/event/views/event_registration_views.xml

```diff
@@ -318,6 +339,29 @@
                     'registration_view_hide_group_by_create_date_week': 1,
                 }
             </field>
+            <field name="search_view_id" ref="view_registration_search_not_specific_to_an_event"/>
+            <field name="help" type="html">
+                <p class="o_view_nocontent_smiling_face">
+                    No Attendees yet!
+                </p><p>
+                    From this dashboard you can report, analyze and detect trends regarding your event registrations.
+                </p>
+            </field>
+        </record>
+
+	    <record id="act_event_registration_stat_from_event" model="ir.actions.act_window">
```

Comment:
According to guidelines this should be ``event_registration_action_stats_from_event`` (model, purpose, details).